import_export/admin.py

```diff
@@ -451,6 +451,12 @@ def write_to_tmp_storage(self, import_file, input_format):
         tmp_storage.save(data)
         return tmp_storage
 
+    def add_data_read_fail_error_to_form(self, form, e):
+        form.add_error('import_file',
+                       _(f"'{type(e).__name__}' encountered while trying to read file. "
+                         "Ensure you have chosen the correct format for the file. "
+                         f"{str(e)}"))
```

Comment:
You can’t have format strings as translation strings.  At runtime, the string used to loop up the translation will look like `'ValueError' encountered while etc etc` and won’t match any msgid in the catalog file.  Translatable strings must look like this:
```python
msg = _("{exception_name} encountered while doing etc etc").format(exception_name=repr(type(e).__name__))
form.add_error('import_file', f'{msg}\n{e}')
```